src/roiextractors/testing.py

```diff
@@ -252,12 +256,13 @@ def check_segmentation_return_types(seg: SegmentationExtractor):
         element_dtypes=floattype,
         shape_max=(np.prod(seg.get_image_size()), 3),
     )
-    _assert_iterable_complete(
-        seg.get_image(),
-        dtypes=(np.ndarray, NoneType),
```

Comment:
Isn't the NoneType here taking care of if seg.get_image() is empty? or what is the function of the loop here?

Reply:
for `seg.get_image()` the name by default is "correlation", if "correlation" is not in `get_images_dict()` then this errors out before that.

Reply:
In `ExtractSegmentationExtractor` instead of correlation or mean image we have other images: https://github.com/catalystneuro/roiextractors/blob/master/src/roiextractors/extractors/schnitzerextractor/extractsegmentationextractor.py#L243

The intention here is if there is no correlation image, we should check the images that are actually in the segmentation

Reply:
I see, that make sense. Thanks.